src/form/acr.jl

```diff
@@ -145,3 +167,192 @@ q[t_idx] == -(b+b_to)*v[t_bus]^2 - (-b*tr+g*ti)/tm*(v[t_bus]*v[f_bus]*cos(t[f_bu
 function constraint_ohms_tp_yt_to(pm::PMs.GenericPowerModel{T}, n::Int, c::Int, f_bus, t_bus, f_idx, t_idx, g, b, g_to, b_to, tr, ti, tm) where T <: PMs.AbstractACRForm
     constraint_ohms_tp_yt_from(pm, n, c, t_bus, f_bus, t_idx, f_idx, g, b, g_to, b_to, tr, ti, tm)
 end
+
+
+function constraint_load_power_setpoint_wye(pm::PMs.GenericPowerModel{T}, nw::Int, cnd::Int, load_id::Int, pd::Real, qd::Real) where T <: PMs.AbstractACRForm
+    JuMP.@constraint(pm.model, PMs.var(pm, nw, cnd, :pd, load_id)==pd)
+    JuMP.@constraint(pm.model, PMs.var(pm, nw, cnd, :qd, load_id)==qd)
+end
+
+
+function constraint_load_power_prop_vm_wye(pm::PMs.GenericPowerModel{T}, nw::Int, cnd::Int, load_id::Int, load_bus_id::Int, cp::Real, cq::Real) where T <: PMs.AbstractACRForm
+    pd = PMs.var(pm, nw, cnd, :pd, load_id)
+    qd = PMs.var(pm, nw, cnd, :qd, load_id)
+    vr = PMs.var(pm, nw, cnd, :vr, load_bus_id)
+    vi = PMs.var(pm, nw, cnd, :vi, load_bus_id)
+    JuMP.@NLconstraint(pm.model, pd==cp*sqrt(vr^2+vi^2))
+    JuMP.@NLconstraint(pm.model, qd==cq*sqrt(vr^2+vi^2))
+end
+
+
+function constraint_load_power_prop_vmsqr_wye(pm::PMs.GenericPowerModel{T}, nw::Int, cnd::Int, load_id::Int, load_bus_id::Int, cp::Real, cq::Real) where T <: PMs.AbstractACRForm
+    pd = PMs.var(pm, nw, cnd, :pd, load_id)
+    qd = PMs.var(pm, nw, cnd, :qd, load_id)
+    vr = PMs.var(pm, nw, cnd, :vr, load_bus_id)
+    vi = PMs.var(pm, nw, cnd, :vi, load_bus_id)
+    JuMP.@NLconstraint(pm.model, pd==cp*(vr^2+vi^2))
+    JuMP.@NLconstraint(pm.model, qd==cq*(vr^2+vi^2))
```

Comment:
This looks like a bus shunt, or am I missing something?

Reply:
Yes, the 'proportional_vmsqr' or 'constant impedance' loads are equivalent to bus shunts.